docs/containers/debug-netcore.md

```diff
@@ -19,8 +19,28 @@ MetaDescription: Debug a .NET app running in a Docker container, using Visual St
 
 ## Walkthrough
 
-1. If needed, create a .NET project with `dotnet new`.
-1. Open the project folder in VS Code.
+- If needed, create a .NET project with `dotnet new`.
+- Open the project folder in VS Code.
+- You are prompted with whether to build your image using a Dockerfile or the [.NET SDK](https://www.microsoft.com/net/download). If you want a simple container debug experience without a Dockerfile or if you are more familiar with MSBuild, choose **Use .NET SDK**. If you're more comfortable working with Dockerfiles, choose **Use a Dockerfile**.
+
+### .NET SDK Container Build (Debug without `Dockerfile`)
+
+// TODO: add context
+
+1. Press `kb(workbench.action.debug.start)` or choose **Start Debugging** from the **Run** menu.
```

Comment:
Does `kb(workbench.action.debug.start)` render prettier on the docs site?

Reply:
Yep, this renders as the correct keyboard shortcut for the readers platform, with a hover for the other two supported.

![image](https://github.com/microsoft/vscode-docs/assets/12818376/a0ad90ec-5063-4b12-840a-b1523dbdc6bf)

Not so interesting for `F5` but nice for Ctrl+* on Windows/Linux versus Cmd+* on macOS

Reply:
Nice!